client/rest-high-level/src/main/java/org/elasticsearch/client/RequestConverters.java

```diff
@@ -652,6 +653,19 @@ static Request setIndexLifecyclePolicy(SetIndexLifecyclePolicyRequest setPolicyR
         return request;
     }
 
+    static Request removeIndexLifecyclePolicy(RemoveIndexLifecyclePolicyRequest setPolicyRequest) {
```

Comment:
nit: can we named the variable `removeLifecycleRequest`?

Reply:
Oops!  Good catch, thanks.